lib/swaps/Swaps.ts

```diff
@@ -335,6 +335,9 @@ class Swaps extends EventEmitter {
     try {
       route = await swapClient.getRoute(makerUnits, destination, makerCurrency);
     } catch (err) {
+      if (err.message?.includes('insufficient local balance')) {
```

Comment:
This is an lnd specific message so it should detected and handled within the LndClient class in the `getRoute` method. I think ideally the LndClient will throw a specific error type (you can add it to swaps/errors.ts) indicating that there's insufficient balance, and then that error should be checked for here. That way Connext/other swap clients can throw the same error later on.